android/app/src/main/java/com/reactnativenavigation/views/LightBox.java

```diff
@@ -54,6 +54,15 @@ private void createContent(final Context context, LightBoxParams params) {
         lightBox.setBackgroundColor(params.backgroundColor.getColor());
         lightBox.addView(content, lp);
 
+        if (params.tapBackgroundToDismiss) {
+            lightBox.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View v) {
+                    hide();
```

Comment:
You better call `destroy()` which dismisses the lightBos (Dialog) and unmounts the react view.